bucket/gitql.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "0.6.0",
+    "description": "SQL like lanuage to perform queries on .git files",
+    "homepage": "https://github.com/amrdeveloper/gql",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/AmrDeveloper/GQL/releases/download/0.6.0/gql-x86_64-windows.exe",
+            "hash": "0b7dc2b055204d12eae9f8e29f2755d7df1a63f527af10340a489b42328b6e3f"
+        }
+    },
+    "pre_install": "Rename-Item \"$dir\\gql-x86_64-windows.exe\" \"$dir\\gitql.exe\"",
+    "bin": "gitql.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/AmrDeveloper/GQL/releases/download/$version/gql-x86_64-windows.exe"
+            }
+        }
```

```suggestion
        },
        "hash": {
            "url": "$baseurl/$basename.sha256"
        }
```

Link to checksums files if available.

---

CI: Property 'hash' has not been defined and the schema does not allow additional properties

---

> CI: Property 'hash' has not been defined and the schema does not allow additional properties

Sorry, my fault while inserting.
`hash` section should be under `autoupdate`, on same level as `architecture`.

---

No problem, i fixed it :D

Thanks for suggesting

---

Done it passed @aliesbelik